docs/latest/tutorial/tutorial-scaffolding.md

```diff
@@ -0,0 +1,354 @@
+via `mainWindow.webContents.openDevTools()` in the main process. If you want to use VS Code,
+keep reading!
+
+The renderer has access to the web APIs. That means you can use the same JavaScript APIs and
+tooling you use for typical front-end development, such as using [`webpack`][webpack] to bundle
+and minify your code or [React][react] to manage your user interfaces.
+
+:::tip
+You can find more information about this subject in the [Boilerplates and CLIs].
+:::
+
+### Debugging from VS Code
+
+If you want to debug your application using VS Code instead of using the terminal you have to
+keep in mind there are 2 processes to attach at this moment: the main and renderer processes.
+
+We are going to create a configuration file that allow us to debug both at the same time:
+
+1. Create a new folder `.vscode` in the root of your project.
+1. Create a new file `launch.json` with the following content:
+
+```json title='.vscode/launch.json'
+{
+  "version": "0.2.0",
+  "compounds": [
+    {
+      "name": "Main + renderer",
+      "configurations": ["Main", "Renderer"],
+      "stopAll": true
+    }
+  ],
+  "configurations": [
+    {
+      "name": "Renderer",
+      "port": 9222,
+      "request": "attach",
+      "type": "pwa-chrome",
+      "webRoot": "${workspaceFolder}"
+    },
+    {
+      "name": "Main",
+      "type": "pwa-node",
+      "request": "launch",
+      "cwd": "${workspaceFolder}",
+      "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron",
+      "windows": {
+        "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron.cmd"
+      },
+      "args": [".", "--remote-debugging-port=9222"],
+      "outputCapture": "std",
+      "console": "integratedTerminal"
+    }
+  ]
+}
+```
+
+<!-- markdownlint-disable MD029 -->
+
+3. The option `Main + renderer` will appear when you select `Run and Debug`
+from the sidebar, allowing you to set breakpoints and inspect all the variables among
+other things in both the main and renderer processes.
+<!-- markdownlint-enable MD029 -->
+
+What we have done in the `lauch.json` file is to create 3 configurations:
+
+- `Main` is use to start the main process and also expose the port 9222 for remote debugging
+  (`--remote-debugging-port=9222`). This is the port that we will use to attach the debugger
+  for the `Renderer`. Because the main process is a Node.js process, the type is set to
+  `pwa-node` (`pwa-` is the prefix that tells VS Code to use the latest JavaScript debugger).
+- `Renderer` is used to debug the renderer process. It tries to connect to the port 9222.
+  Because the renderer is a web process, the debugger that we use is `pwa-chrome`.
+- `Main + renderer` is a [compound task] that executes the previous ones simulatenously.
+
+:::caution
+Because we are attaching to a process that exists already in the `Renderer` configuration
+(`"request": "attach"`) there is a high chance that you will not be able to set a breakpoint
```

Comment:
I think we should explain the `request: attach` thing above. I'm not too familiar with it myself. Does it mean that we're attaching to the existing remote debugging port at 9222 created when running the "Main" configuration?

Does this also work with multiple renderer processes?

Reply:
> Does it mean that we're attaching to the existing remote debugging port at 9222 created when running the "Main" configuration?

Yes, haven't tried the multiple renderer process.
I'll make it clearer.

Reply:
I've updated the `request: "attach"` in case you want to take another look